### PR TITLE
Create translation-plugin

### PR DESCRIPTION
**Pull request by PlayerCoder1**
Translates everything in the chatbox from NPC dialogs to Players talking on the side panel for now it's English to Spanish but I'm going to add more langages in the futur

### CODE REVIEW

**Comment by Felanbird** [comment]
Github branch names do not matter, only the file name that you are trying to add to this repo, you didn't need to open a new PR

**Comment by PlayerCoder1** [comment]
> Github branch names do not matter, only the file name that you are trying to add to this repo, you didn't need to open a new PR

Ah damn my bad :(

**Comment by PlayerCoder1** [comment]
Update on the plugin, added more langages ES/IT/FR/NL you can directly change it from the Panel

**Comment by PlayerCoder1** [comment]
What went wrong with the checks?

**Comment by Nightfirecat** [comment]
You can see for yourself in the build logs: https://github.com/runelite/plugin-hub/actions/runs/5404888845/jobs/9826688261?pr=4542
[Your plugin is using dependencies](https://github.com/PlayerCoder1/translation-plugin/blob/1db58cc94b4d4d5b9d37236ea013b3f875e03ad6/build.gradle#L25) which we don't have in our verification template. We [already have two other versions of json pinned there](https://github.com/runelite/plugin-hub/blob/e8388eb17b584cd0ba30c2986ea071110c33dfb1/package/verification-template/gradle/verification-metadata.xml#L794), I recommend you use one of those.

**Comment by PlayerCoder1** [comment]
> You can see for yourself in the build logs: https://github.com/runelite/plugin-hub/actions/runs/5404888845/jobs/9826688261?pr=4542 [Your plugin is using dependencies](https://github.com/PlayerCoder1/translation-plugin/blob/1db58cc94b4d4d5b9d37236ea013b3f875e03ad6/build.gradle#L25) which we don't have in our verification template. We [already have two other versions of json pinned there](https://github.com/runelite/plugin-hub/blob/e8388eb17b584cd0ba30c2986ea071110c33dfb1/package/verification-template/gradle/verification-metadata.xml#L794), I recommend you use one of those.

Sorry very new to github, I think I updated what needed to be updated

**Comment by Nightfirecat** [comment]
Your build is still failing because in runelite-plugin.properties you've set `plugins=com.PlayerCoder1.translation-plugin`, but there is no `src/main/java/com/PlayerCoder1/translation-plugin` class. It looks like it should be changed to `plugins=com.PlayerCoder1.TranslationPlugin`

**Comment by PlayerCoder1** [comment]
> Your build is still failing because in runelite-plugin.properties you've set `plugins=com.PlayerCoder1.translation-plugin`, but there is no `src/main/java/com/PlayerCoder1/translation-plugin` class. It looks like it should be changed to `plugins=com.PlayerCoder1.TranslationPlugin`

Ok sorry I didn't even see the build, the repo is now updated

File: plugins/translation-plugin [hunk 1]
```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/PlayerCoder1/translation-plugin.git
+commit=7084191592c89454cde1b558c604c7c5283ab92f
```
Comment (Nightfirecat) [inline plugins/translation-plugin:2]: This plugin needs a `warning` for its integration with a third-party service:
```suggestion
commit=7084191592c89454cde1b558c604c7c5283ab92f
warning=This plugin submits all observed chat messages and your IP address to a server not controlled or verified by the RuneLite developers.
```

**Review by Nightfirecat — COMMENTED** [review]
Code LGTM

**Pull request merged**